### PR TITLE
APPLICS-93: Welsh translation - All projects - static content

### PR DESCRIPTION
**Pull request by christopher-semmens**
## Describe your changes

https://pins-ds.atlassian.net/browse/APPLICS-93?atlOrigin=eyJpIjoiYTlkOWEwZmU2YjVmNGIxODk1NmMxOGQ5MWE3NjRjYTAiLCJwIjoiaiJ9

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: packages/forms-web-app/src/pages/project-search/_translations/cy.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	"linkText1": "Lawrlwythwch dabl sy'n cynnwys rhestr lawn o bob un o'r {{-totalApplications}} o brosiectau (CSV)",
+	"accessibilityText1": "Chwilio am brosiect",
+	"heading2": "Canlyniadau wedi'u hidlo",
+	"paragraph2": "Yn dangos {{-from}} i {{-to}} o {{-total}} brosiectau.",
+	"noResultsFound": {
+		"paragraph1": "Ni ddaethpwyd o hyd i unrhyw ganlyniadau sy'n cyfateb i'ch termau chwilio a'ch dewisiadau hidlo.",
+		"paragraph2": "A hoffech chi glirio'ch chwiliad a'ch dewisiadau hidlo i weld yr holl brosiectau yn lle hynny?",
+		"linkText1": "Clirio chwiliad a dewisiadau hidlo"
+	},
+	"filterLabels": {
```
Comment (cprieststephens) [inline packages/forms-web-app/src/pages/project-search/_translations/cy.json:17]: Are there Welsh translations to be added here?

**Review by cprieststephens — COMMENTED** [review]
Just added one question about translations otherwise looks good to me.

File: packages/forms-web-app/src/pages/project-search/_translations/cy.json (hunk 1 above)
Comment (christopher-semmens) [reply packages/forms-web-app/src/pages/project-search/_translations/cy.json:17]: @cprieststephens Yes, we are still waiting on those translations.

**Review by cprieststephens — APPROVED** [review]

**Pull request merged**